controllers/flp/flp_ingest_reconciler.go

```diff
@@ -32,58 +28,54 @@ type ingestOwnedObjects struct {
 	prometheusRule *monitoringv1.PrometheusRule
 }
 
-func newIngesterReconciler(cmn *reconcilers.Instance) *flpIngesterReconciler {
+func newIngesterReconciler(cmn *reconcilers.Instance) *ingesterReconciler {
 	name := name(ConfKafkaIngester)
-	owned := ingestOwnedObjects{
-		daemonSet:      &appsv1.DaemonSet{},
-		promService:    &corev1.Service{},
-		serviceAccount: &corev1.ServiceAccount{},
-		configMap:      &corev1.ConfigMap{},
-		roleBinding:    &rbacv1.ClusterRoleBinding{},
-		serviceMonitor: &monitoringv1.ServiceMonitor{},
-		prometheusRule: &monitoringv1.PrometheusRule{},
-	}
-	cmn.Managed.AddManagedObject(name, owned.daemonSet)
-	cmn.Managed.AddManagedObject(name, owned.serviceAccount)
-	cmn.Managed.AddManagedObject(promServiceName(ConfKafkaIngester), owned.promService)
-	cmn.Managed.AddManagedObject(RoleBindingName(ConfKafkaIngester), owned.roleBinding)
-	cmn.Managed.AddManagedObject(configMapName(ConfKafkaIngester), owned.configMap)
+	rec := ingesterReconciler{
+		Instance:       cmn,
+		daemonSet:      cmn.Managed.NewDaemonSet(name),
+		promService:    cmn.Managed.NewService(promServiceName(ConfKafkaIngester)),
+		serviceAccount: cmn.Managed.NewServiceAccount(name),
+		configMap:      cmn.Managed.NewConfigMap(configMapName(ConfKafkaIngester)),
+		roleBinding:    cmn.Managed.NewCRB(RoleBindingName(ConfKafkaIngester)),
+	}
```

Comment:
The new Managed API make this part more readable, this is nice.

Reply:
yeah I don't know why I didn't do that earlier :)